src/desktop/apps/artist/components/cta.js

```diff
@@ -31,7 +32,13 @@ const send = {
 }
 
 if (sd.ARTIST_PAGE_CTA_ENABLED && sd.ARTIST_PAGE_CTA_ARTIST_ID) {
-  metaphysics(send).then(({ artist }) => {
-    cta(new Artist(artist))
+  metaphysics(send).then(({ artist: artistData }) => {
+    const artist = new Artist(artistData)
```

Comment:
This is a legit diff, I added this snippet to account for: https://github.com/artsy/force/blob/f1f35ef971ca3572696aa56306a09b923990e99b/src/desktop/apps/artist/client/cta.coffee (we're reusing the old CTA component overlaid on the new artist page), which we were using directly.